### ISSUE TITLE
No translation of admin template's html lang attribute

### ISSUE THREAD

**Issue by webdecker** [processwire/processwire-issues#1273]
### Short description of the issue

The admin templates let the html lang attribute value to be translated. So 'en' becomes for example 'de'.
But because of this translation and the translation mechanism of admin page's h1 headlines, on the page edit page of a page named/titled 'en' the headline is 'de' - very confusing.

### Expected behavior

1) The admin page's h1 headline should remain 'de'.
I guess the headline translation is used for multi language websites using field translation(?). But if you don't, no translation should be applied (or do I miss some important point here...?).
2) The html lang attribute value should be something like 
echo ($user->language->name == 'default' ? 'en' : $user->language->name)

### Actual behavior

html lang attribute value is rendered as translation of 'en'.

### Suggestion for a possible fix

See "Expected behavior" (2). If html lang attribute is rendered without translation then the corresponding item in the translation jsons could be removed and the actual problem would be solved.
"Expected behavior" (1) is propably an issue of its own...?

### Steps to reproduce the issue

1. Login to pw backend of a website wich has a langauge pack (e.g. 'de') installed
2. Below any page create a new page, set title 'en' and save
3. The h1 headline is 'de' (according to language pack...)

### Setup/Environment

PW 3.0.165

### Remark 
I came to this issue because I plan to create a mutli language website where the language is derived by a "language prefix" of the url like /en/somepage/ or /de/somepage/. Hence I created a page named 'en', ...

**Comment by ryancramerdesign** [comment]
Thanks @webdecker I've pushed a fix for this issue. This appears to be an isolated case for this language attribute. It has to remain translatable with the language pack since languages are pages, which can have any name the user chooses (which may or may not be the same as an HTML language attribute). The simplest solution here seems to be just preventing 'en' from getting the default admin file context applied when it comes to translation.

**Comment by netcarver** [comment]
@webdecker Hi Ingmar, could you verify that this has been fixed and either update or close this issue please.

**Comment by webdecker** [comment]
Thank you for dealing with this issue! I tested it with 3.0.168 and de language pack and I see the fix in AdminThemeFramework::getHeadline().
So the "core problem" has been solved, thanks!

I still wonder whether this problem may appear again with some other language codes - think of "it"... Anyway,,,

But still I would suggest, that the admin theme templates don't translate the html lang attribute but show the user's language's name.  Because that is what the html page shows, what the basis of the current translation is. Wouldn't this be a bit more "elegant" or "correct"...?
